ontology/rxno.md

```diff
@@ -23,6 +23,7 @@ products:
 activity_status: active
 repository: https://github.com/rsc-ontologies/rxno
 preferredPrefix: RXNO
+domain: biochemistry
```

Comment:
@batchelorc thoughts?

Reply:
nooo... this is definitely chemistry rather than biochemistry

Reply:
```suggestion
domain: chemistry and biochemistry
```